packages/ref-metrics-ilamb3/pyproject.toml

```diff
@@ -0,0 +1,39 @@
+[project]
+name = "cmip_ref_metrics_ilamb3"
+version = "0.1.5"
+description = "ilamb3 metrics provider for the CMIP Rapid Evaluation Framework"
+readme = "README.md"
+authors = [
+    { name = "Nathan Collier", email = "nathaniel.collier@gmail.com" }
+]
+requires-python = ">=3.10"
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "Operating System :: OS Independent",
+    "Intended Audience :: Science/Research",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Topic :: Scientific/Engineering",
+]
+dependencies = [
+    "ilamb3"
```

Comment:
There is still a dependecy on `cmip_ref_core`. It might just be for the type hints though